tests/test_toa_writer.py

```diff
@@ -49,6 +49,25 @@ def test_roundtrip_topo_toa_Tempo2format(self):
         assert np.abs(ts.table["mjd"][0] - ts2.table["mjd"][0]) < 1.0e-15 * u.d
         assert np.abs(ts.table["tdb"][0] - ts2.table["tdb"][0]) < 1.0e-15 * u.d
 
+    def test_commenting_toas(self):
+        # Create a barycentric TOA
+        t1time = Time(58534.0, 0.0928602471130208, format="mjd", scale="utc")
+        t1 = toa.TOA(t1time, obs="gbt", freq=0.0)
+        t2 = toa.TOA(t1time, obs="gbt", freq=0.0)
+        ts = toa.get_TOAs_list([t1, t2], ephem="DE421")
+        assert ts.ntoas == 2
+        ts.table[1]["flags"]["ignore"] = True  # flag one TOA
```

Comment:
Maybe also check the default behaviour? That is, if the default is "cut", maybe check that that happens when you pass no argument?

Reply:
I'm fine doing that, but checking for `ignore` tests for the default behavior (since `ignore` is a string like `cut`, just not the default -- and I don't think we need to test that Python's optional arguments work) as well as the user choosing something besides the default string.  Seems like a waste of a (small amount) of test time.

Reply:
We had a bug earlier this year where the test suite never executed a method with the default value of its argument, so that default value had the wrong units and we never noticed. Tests also detect when the public API changes, and the default value is part of that public API, so it's a good idea to check that.